Jenkinsfile.merge

```diff
@@ -0,0 +1,47 @@
+pipeline {
+    agent {
+        label 'testintegration'
+//        docker {
+//            image 'docker.io/manics/omero-build-gradle:latest'
+//        }
+    }
+
+    parameters {
+        choice(
+            name: 'STATUS',
+            choices: ['success-only', 'no-error', 'none'],
+            description: 'PR check status'
+        )
+        string(
+            name: 'MERGE_OPTIONS',
+            defaultValue: '-vvv --no-ask --reset --comment',
+            description: 'scc merge options'
+        )
+    }
+
+    environment {
+        // These environment variables can be overridden in
+        // Manage Jenkins → Configure System → Global properties → Environment variables
+
+        // MERGE_PUSH_BRANCH
+        // MERGE_GIT_USER
+
+        BASE_REPO = 'omero-build.git'
+    }
+
+    stages {
+        stage('Merge') {
+            steps {
+                // build is in .gitignore so we can use it as a temp dir
+                sh 'mkdir build'
+                sh 'cd build && curl -sfL https://github.com/manics/build-infra/archive/omero-build.tar.gz | tar -zxvf -'
```

Comment:
This will need to be updated when https://github.com/ome/build-infra/pull/1 is merged

Reply:
good to go. also remove the commented out blocks while you're at it?